faq/faq.md

```diff
@@ -0,0 +1,351 @@
+copyright and license tag.
+
+If you really want to exclude a file, consider using the
+[CC0](https://creativecommons.org/publicdomain/zero/1.0/) license for this
+file.  By doing this, you put the file in the public domain, or your country's
+equivalent.
+
+There is one exception: License files are automatically excluded from compliance
+testing.
+
+<!-- FIXME: Link to uncopyrightable question and DEP5 question -->
+
+## Do you support a version control system other than Git? {#no-git}
+
+Currently we do not, but please [get in
+touch](https://github.com/fsfe/reuse-tool/issues).
+
+# Licensing and compliance {#licensing}
+
+## Which licenses are compatible? {#compatible-licenses}
+
+## Is there a standard format for declaring copyright? {#standard-copyright}
+
+Copyright year copyright holder email
+
+## Which years do I include in the copyright statement? {#years-copyright}
+
+## Do I need to include both GPL-3.0-or-later and GPL-3.0-only in my repository? {#gpl-plus}
+
+Members of the GPL family are listed separately in the SPDX License List as
+-only and -or-later, even though the license texts are identical. If you have
+code under only one of these licenses, we recommend that you only include that
+one license.
+
+If you have code under both licenses, we recommend that you include both
+licenses separately.
+
+If you do not want to include both licenses separately (e.g., for space
+reasons), you can create the file `LICENSES/GPL-3.0.txt` and add the following
+header:
+
+```
+Valid-License-Identifier: GPL-3.0-only
+Valid-License-Identifier: GPL-3.0-or-later
+License-Text:
+
+[license text]
+```
+
+## What are license exceptions and what do I do with them? {#license-exceptions}
+
+## Do I use SPDX-Copyright, Copyright, or ©? {#copyright-symbol}
+
+## Which files are copyrightable? {#what-is-copyrightable}
+
+All files that are original works of authorship are copyrightable. In essence,
+if someone sat down typing their own original thoughts on a keyboard, then that
+person holds copyright over the output. Common examples are source code,
+documentation, audio, and video.
+
+There are some edge cases, however. For instance, the program `print("Hello,
+REUSE!")` probably does not meet the threshold of originality. Similarly, data
+files and configuration files may not meet that threshold either.
+
+## What to do with uncopyrightable files? {#uncopyrightable}
+
+There are two things that you can do with such a file to make sure that its
+copyright and licensing is recorded. The first option is to simply use your
+regular copyright and license header for this file. There is nothing that stops
+you from claiming copyright over your own works.
+
+The alternative is to waive your copyright by using the
+[CC0](https://creativecommons.org/publicdomain/zero/1.0/) license.
+
+It is important to note that you can only do this for your own works. If the
+file was authored by someone else, you must declare their copyright and license
+in the header.
```

Comment:
Technically true, yes. Could become problematic with what to do with config and temp files etc. that nobody claims copyright over (or are even machine-generated). We don’t want people to keep files in a limbo.

Reply:
Don't really know how to solve that neatly :(